# task2:
# Написать программу которая принимает число от пользователя, и затем делит на него число 100.
# Отловить ошибку приведения полученной от пользователя строки к целому и ошибку деления на 0.
# Отлавливать ошибки надо каждую в одном блоке except и выводить соответствующее сообщение,
# которое передается в объекте ошибки.

number = input('Введите число: ')
try:
    result = 100 / int(number)
except ZeroDivisionError:
    print('Ошибка деления на ноль')
except ValueError:
    print('Введено не число')
else:
    print(result)

# Результат:
# $ python3 try2.py
# Введите число: 0
# Ошибка деления на ноль
# $ python3 try2.py
# Введите число: fdg
# Введено не число
# $ python3 try2.py
# Введите число: 2
# 50.0
